flaps/flaps.go

```diff
@@ -68,14 +69,9 @@ func NewWithOptions(ctx context.Context, opts NewClientOpts) (*Client, error) {
 	}
 
 	if opts.DialContext != nil {
-		orgSlug, err := resolveOrgSlugForApp(ctx, opts.AppCompact, opts.AppName)
-		if err != nil {
-			return nil, fmt.Errorf("failed to resolve org for app '%s': %w", opts.AppName, err)
-		}
-
```

Comment:
This code was actually already being called by the caller in `flyctl` so this removes the duplication. The result of the code in `flyctl` is set to `opts.OrgSlug`.